src/Jackett.Common/Indexers/SpeedCD.cs

```diff
@@ -90,20 +92,31 @@ public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken
 
         private async Task DoLogin()
         {
+            // first request with username
             var pairs = new Dictionary<string, string> {
-                { "username", configData.Username.Value },
-                { "password", configData.Password.Value },
+                { "username", configData.Username.Value }
             };
-
-            var result = await RequestLoginAndFollowRedirect(LoginUrl, pairs, null, true, null, SiteLink);
+            var result = await RequestLoginAndFollowRedirect(LoginUrl1, pairs, null, true, null, SiteLink);
+            var tokenRegex = new Regex(@"name=\\""a\\"" value=\\""([^""]+)\\""");
+            var matches = tokenRegex.Match(result.Content);
+            if (!matches.Success)
+                throw new ExceptionWithConfigData("Error parsing the login form", configData);
+            var token = matches.Groups[1].Value;
+
+            // second request with token and password
+            pairs = new Dictionary<string, string> {
+                { "a", token },
+                { "b", configData.Password.Value },
+            };
+            result = await RequestLoginAndFollowRedirect(LoginUrl2, pairs, result.Cookies, true, null, SiteLink);
 
             await ConfigureIfOK(result.Cookies, result.Content?.Contains("/browse.php") == true, () =>
             {
                 var parser = new HtmlParser();
                 var dom = parser.ParseDocument(result.Content);
                 var errorMessage = dom.QuerySelector("h5")?.TextContent;
                 if (result.Content.Contains("Wrong Captcha!"))
-                    errorMessage = "Captcha requiered due to a failed login attempt. Login via a browser to whitelist your IP and then reconfigure jackett.";
+                    errorMessage = "Captcha requiered due to a failed login attempt. Login via a browser to whitelist your IP and then reconfigure Jackett.";
```

Comment:
a pity that while you were capitalising the **J** in Jackett you did not spot the misspelling of **required**  ;-D

Reply:
i've pushed a type correction.